LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
This file is required

Reply:
will add back in. That's one accidental deletion

Reply:
done